CMake/switch/devkita64-libnx.cmake

```diff
@@ -23,6 +23,16 @@ set(PORTLIBS_PATH ${DEVKITPRO}/portlibs)
 set(PORTLIBS ${PORTLIBS_PATH}/switch)
 set(CMAKE_FIND_ROOT_PATH ${DEVKITA64} ${LIBNX} ${PORTLIBS})
 
+# Allow FIND_PACKAGE to find SDL2_ttf and other libraries
+set(CMAKE_PREFIX_PATH "${DEVKITPRO}/portlibs/switch/")

```

Comment:
Strange that this is needed. We already have `${PORTLIBS}` in `CMAKE_FIND_ROOT_PATH` (line 24).

Reply:
Thanks! You are right. It is not needed. I must have made a mistake.
Fixed.